rust-toolchain.toml

```diff
@@ -1,4 +1,4 @@
 [toolchain]
-channel = "1.67.0"
+channel = "1.67.1"
```

Comment:
can you revert this change actually 😅 it makes upgrades on the Mina side quite annoying

Reply:
done in https://github.com/o1-labs/proof-systems/pull/1027

Reply:
Sure thing!